tests/python/gaia-ui-tests/gaiatest/apps/search/app.py

```diff
@@ -34,7 +34,7 @@ def go_to_url(self, url):
             self._root_element = self.marionette.find_element(*self._browser_app_locator)
             self._root_element.find_element(*self._url_bar_locator).tap()
 
-        from gaiatest.apps.homescreen.regions.search_panel import SearchPanel
+        from gaiatest.apps.system.regions.search_panel import SearchPanel
```

Comment:
`apps/homescreen/app.py` that import is going to be removed, it's not used anymore.